xdsl/tools/xdsl_tblgen.py

```diff
@@ -521,7 +521,7 @@ def cull_field(js_in: dict[str, Any]) -> dict[str, Any]:
         key: js["!instanceof"][key] for key in ("TypeDef", "AttrDef", "Op", "Dialect")
     }
 
-    print(json.dumps(culled), file=output_file)
+    print(json.dumps(culled, sort_keys=True), file=output_file)
```

Comment:
oh, we can avoid the jq thing completely, right?
```suggestion
    print(json.dumps(culled, sort_keys=True, indent=4), file=output_file)
```

Reply:
Perhaps, I'll try it after seminars